src/reporter.js

```diff
@@ -31,7 +25,7 @@ export default function (message) {
         }
       }
     }, function() {});
-  }
 
-  console.log("AccessLint warnings: ", violations);
+    console.warn(violations);
```

Comment:
Would printing a description above the object as a warning help the user parse through a long list of warnings?

Looking at your GIF in the PR (that was awesome, BTW), I noticed that with 11 warnings it could be a little cumbersome to sort through them since they are all listed as `[Object object]`.